jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -458,7 +471,14 @@ def _image_viewer_update(self, viewer, x, y):
             elif isinstance(viewer, CubevizImageView):
                 arr = image.get_component(attribute).data
                 unit = image.get_component(attribute).units
-                value = self._get_cube_value(image, arr, x, y, viewer)
+                value = self._get_cube_value(
+                    image, arr, x, y, viewer
+                )
+                if self.current_unit is not None and self.current_unit.is_equivalent(unit):
```

Comment:
this seems to be the logic that keeps velocity mouseover working for moment maps.  Just a note that we'll need to rethink this if/when we have velocity display units as well, since we'll then need to have per-viewer image units based on each viewers "physical type" and its corresponding display units 🤔